### ISSUE TITLE
Bugs found during testing

### ISSUE THREAD

**Issue by milancermak** [BibliothecaForAdventurers/realms-react#72]
* [x] Fly to button (top right) doesn't work. Clicking it doesn't take me to the selected Realm, only pops up the side menu of Realm no. 1
* [x] Also using the Fly to button from the Realms menu doesn't seem to work properly. Takes me to a seemingly random place on the map.
* [x] Elements on the map are not interactive. When clicking on an icon, I'd expect the side menu to show me the details of the object. Right now, nothing happens.
* [x] Details button in Crypts menu doesn't do anything.

**Comment by tenoxious** [comment]
should be resolved by #73

**Comment by 0xcodemonkey** [comment]
- [ ]  All dropdown filters and search by ID for the personal items tabs ("Your Realms", "Your Crypts", etc) and favorite items tabs don't work. It does work when you switch to the all items tabs ("All Realms", etc).
- [ ] Details buttons should ideally toggle the side panel both on and off. Right now they only toggle on.
- [ ] Trade page (via icon on the left panel) just says "test". I'm guessing this is expected but if it's not going to be in the initial release, I would hide the Trade icons on the left panel and on the Iron Bank page.

**Comment by tenoxious** [comment]
- All dropdown filters and search by ID for the personal items tabs ("Your Realms", "Your Crypts", etc) and favorite items tabs don't work. It does work when you switch to the all items tabs ("All Realms", etc).
- For all the slider filters (e.g. traits filter for realms): opening the filter dialog, choosing a new slider value to activate the filter, closing the dialog, then reopening the dialog displays the max value instead of the chosen one
should be fixed in the next build

**Comment by ponderingdemocritus** [comment]
- [ ] Realm Rarity ranking is too steep making the filter basically useless as all 7500 Realms are less than 400 points. So there is a 7500 gap of no realms. Do we slope the number? so 90% of the bar is 1-400 ?

**Comment by ponderingdemocritus** [comment]
- [ ] cannot click assets on the map and have sidebar flyout

**Comment by tenoxious** [comment]
pr is up: cannot click assets on the map and have sidebar flyout

**Comment by tenoxious** [comment]
- [ ] pagination state not clearing on filter update

**Comment by ponderingdemocritus** [comment]
- [ ] Side panel auto opening on mobile when a filter is selected. It shouldn't open until a user clicks a realm

**Comment by LordSquiddy** [comment]
- [ ] if you've loaded the Realm before, resources and traits don't update from the previous Realm once you select 'details'

Flow: clicks Realms on the side menu, click 'details' of the second Realm, click back to 'details' of the first Realm and resources/traits remain from the second Realm

**Comment by tenoxious** [comment]
- [ ] Slider filters don't currently handle touch events

**Comment by ponderingdemocritus** [comment]
have all these been completed?

**Comment by tenoxious** [comment]
i think everything except for:
- [ ] Details buttons should ideally toggle the side panel both on and off. Right now they only toggle on.
- [ ] Slider filters don't currently handle touch events

not sure if i agree with first one
touch events we can split it out in to another issue

**Comment by ponderingdemocritus** [comment]
yes I think that the button toggle the entire panel is preferable to me too.
